Sources/UIKitExtensions/UITabBar+FunctionConfigurationSet.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  UITabBarFunctions.swift
+//  ViewConfigurator-iOS
+//
+//  Created by Andrii Pazynych on 27.03.18.
+//  Copyright © 2018 ImagineOn GmbH. All rights reserved.
+//
+
+import UIKit
+public extension ConfigurationSet where Base: UITabBar {
+    
+    func setItems(_ items: [UITabBarItem]?, animated: Bool) -> ConfigurationSet<Base> {
+        return set { ( UITabBar: Base ) in
+            UITabBar.setItems(items, animated: animated)
```

Comment:
This function is only useful for existing UITabBar whose items change should be animated.
The items Property ist writable directly so there is no need for this function.